tmail-backend/apps/distributed/src/main/java/com/linagora/tmail/james/app/DistributedServer.java

```diff
@@ -225,7 +227,9 @@ public class DistributedServer {
         new CassandraMailboxModule(),
         new CassandraDeletedMessageVaultModule(),
         new MailboxModule(),
-        new TikaMailboxModule());
+        new TikaMailboxModule(),
+        new CassandraMailboxQuotaModule(),
```

Comment:
The `CassandraMailboxQuotaLegacyModule` , `CassandraSieveQuotaLegacyModule` were marked "@Deprecated() // To be removed after release 3.9.0",
IMO the install directly module here will be better than  `chooseQuotaModule` (between new and legacy)

Reply:
We need the chooseQuotaModule? How can we migrate to the new quota schema on our current deployed environments without then?

Reply:
I forgot it, we need a migration